build/cloud/README_zh.md

```diff
@@ -25,4 +25,28 @@ for resource in $(ls *.yaml); do kubectl create -f $resource; done
 ```
 
 最后，基于`08-service.yaml.example`，创建一个适用于你集群环境的 service，
-将 cloud hub 暴露到集群外，让 edge core 能够连到。
\ No newline at end of file
+将 cloud hub 暴露到集群外，让 edge core 能够连到。
+
+---
+> 以下仅针对网络环境不能正常下载相应镜像的说明:
+
+在可能有网络问题的情况下,在执行07-deployment.yaml的时候
+deployment中对应的init container会先去
```

Comment:
I can't find any init container in `build/deployment-armv7.yaml`.

Reply:
That's another question. Just for gfw.
at  `build/cloud/07-deployment.yaml` #line 32

This PR does three things:
add describe about build directory at `build/README_zh.md`
add tip for create for 07-deployment.yaml  at `build/cloud/README_zh.md`
add arm-based deployment sample at `build/deployment-armv7.yaml`

Reply:
oops, I got it. Thanks!

BTW, do you mind squashing the commits?

Reply:
No , I don't mind.